frame/staking/src/weights.rs

```diff
@@ -165,13 +167,17 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Staking Validators (r:2 w:0)
 	// Storage: Staking CurrentEra (r:1 w:0)
 	// Storage: Staking CounterForNominators (r:1 w:1)
+	// Storage: Staking Bonded (r:1 w:0)
```

Comment:
This looks wrong - we pass in the controller account so shouldn't need to look it up

Reply:
This is called whenever we call `weight_of_fn` (see in `fn slashable_balance_of`)

Reply:
Ah yeah - a bit unfortunate. Only work around I see if we where to store both the stash and controller account ids in the node